knative-sandbox-OWNERS_ALIASES

```diff
@@ -183,21 +190,26 @@ aliases:
   - zhanggbj
   net-certmanager-approvers:
   - ZhiminXiang
+  - tcnghia
```

Comment:
was this intentional?

Reply:
that's a result of running `update-codegen.sh` which hasn't run for a while after other changes to peribolos were merged. This one in particular was introduced [here](https://github.com/knative/community/commit/f0ee0738a1656dd3a9f4002a9e05ff34cac25266#diff-48ae180ea1cda2986f85c0177aae1f359defa7eb1f879a0789bb4ebaf85e3faaR216)